src/Module.Server/Balancing/WeightHelpers.cs

```diff
@@ -19,4 +19,9 @@ internal static float ComputeTeamWeight(List<WeightedCrpgUser> team, float p = M
         var clangroupTeam = MatchBalancingHelpers.SplitUsersIntoClanGroups(team);
         return clangroupTeam.Sum(c => c.Weight());
     }
+    internal static float ComputeTeamAbsWeight(List<WeightedCrpgUser> team, float p = MatchBalancer.PowerParameter)
```

Comment:
You should probably have a warning here.

Reply:
what warning?